app/build.gradle.kts

```diff
@@ -17,7 +17,7 @@ dependencies {
         exclude(group = "com.exactpro.th2", module = "cradle-cassandra")
     }
 
-    implementation(platform("org.eclipse.jetty:jetty-bom:11.0.15")) {
+    implementation(platform("org.eclipse.jetty:jetty-bom:11.0.20")) {
```

Comment:
note: Instead of updating the Jetty bom we should update Javalin dependencies

Reply:
Should we use 5.6.5 version ?

Reply:
I think, yes. This is the latest version before the next major release

Reply:
artifacts related to the "io.javalin.community.openapi" grop don't have "5.6.5" version, I have updated them to "5.6.4"
@OptimumCode could you please check